src/test/resources/application-test.properties

```diff
@@ -0,0 +1,13 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+spring.jpa.open-in-view=false
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create-drop
+
+server.servlet.context-path=/api
```

Comment:
did you check if it's going to work the same if you won't put it in test properties?